handler.cpp

```diff
@@ -25,18 +38,38 @@ RequestHandler::Status StaticHandler::Init(const std::string& uri_prefix, const
     return RequestHandler::PASS;
 }
 
-// TODO
+// TODO: url is empty now, might need to change Init? or somehow modify config to have url in it. Also for files that cannot be found, NotFoundHandler has to be invoked. How do we do that?
 RequestHandler::Status StaticHandler::HandleRequest(const Request& request, Response* response) {
+    /*std::cout << "========= Handling Static =========" << std::endl;
+    std::string abs_path = get_exec_path() + "/public" + get_path_from_url(url);
+    std::cout << "Serving file from: " << abs_path << std::endl;
```

Comment:
I do not think this code is valid anymore, but I will take care of static handling so don't worry about it. Just wanted to point that out now.